backend/tests/test_idle.py

```
@@ -374,6 +380,7 @@ async def inner(scope, receive, send):
                 await Response("ok")(scope, receive, send)
 
             client.app.state.groups.app_for = lambda name: inner
+            client.app.state.groups.mounted_members = lambda name: {srv["slug"]: sid}
```

Ruff (0.16.3)

[warning] Unused lambda argument: `name`

(ARG005)